src/for_mathlib/topological_structures.lean

```diff
@@ -178,58 +176,58 @@ instance topological_add_group.to_uniform_space : uniform_space G :=
                                                           end
             ... ↔  (∀ p : G × G, δ p ∈ U → p ∈ S' x) : by simp[δ, S' x]
             ... ↔ δ⁻¹'U ⊆ S' x : set.preimage_subset_iff,
-            
+
             cc } },
-      cc 
+      cc
     end,}
 
 variable {G}
-lemma uniformity_eq_vmap_nhds_zero : uniformity = vmap δ (nhds (0 : G)) := rfl
+lemma uniformity_eq_comap_nhds_zero : uniformity = comap δ (nhds (0 : G)) := rfl
 
-instance topological_add_group_is_uniform : uniform_add_group G := 
+instance topological_add_group_is_uniform : uniform_add_group G :=
```

Comment:
Similarly it would be nice to tell github "also ignore any changes which are of the form vmap->comap" ;-)